dnf-behave-tests/features/repoquery-unsatisfied.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Test for dnf repoquery --unsatisfied
```

Comment:
I think the file should be at `features/repoquery/unsatisfied.feature` instead of `features/repoquery-unsatisfied.feature`.

Reply:
Fixed.